### ISSUE TITLE
Exception for `-webkit-tap-highlight-color: rgba(0,0,0,0);`

### ISSUE THREAD

**Issue by pytkin** [ben-eb/postcss-colormin#1]
Input:
```css
-webkit-tap-highlight-color: rgba(0,0,0,0);
```
Output:
```css
-webkit-tap-highlight-color: transparent;
```

Unfortunately mobile safari not support color names or hex values for `-webkit-tap-highlight-color` when you want [remove tap highlights](https://css-tricks.com/snippets/css/remove-gray-highlight-when-tapping-links-in-mobile-safari/).

From [Apple's docs](https://developer.apple.com/library/safari/documentation/AppleApplications/Reference/SafariCSSRef/Articles/StandardCSSProperties.html#//apple_ref/doc/uid/TP30001266-_webkit_tap_highlight_color): This property obeys the alpha value, if specified. If you don’t specify an alpha value, Safari on iOS applies a default alpha value to the color. To disable tap highlighting, set the alpha value to 0 (invisible). If you set the alpha value to 1.0 (opaque), the element is not visible when tapped.

**Comment by ben-eb** [comment]
OK cool, thanks for the clarification. :smile:

**Comment by ben-eb** [comment]
Fixed in `1.2.3`.

**Comment by pytkin** [comment]
@ben-eb :+1: cool
